content/next_steps/next_steps_2.md

```diff
@@ -0,0 +1,18 @@
++++
+title = "Visualise your pipeline data"
+weight = 2
+post = ""
++++
+
+#### **Visualise your pipeline data**
```

Comment:
Recommend removing this line, as no other pages has it and is just repeating the page title.